keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -3,703 +3,832 @@
-	+ add the access privileges to the `Bucket`.
-	+ return the credentials to the workload owning the `BucketAccessRequest`.
-+  Existing User and existing Bucket: In this scenario, the user account has access policy defined on the `Bucket`.  The existing user privileges in the backend may conflict with the privileges that the user is requesting.
-	+ FAIL, if existing access policy is different from the requested policy.
-	+ if the existing privileges match the requested privileges, return the credentials to the workload owning the `BucketAccessRequest`.
-+ A Service Account specified and the cloud platform identity integration maps Kubernetes ServiceAccounts to the account in the backend storage system. No need to create credentials here.
-	
-Upon success, the `BucketAccess` instance is ready and the app workload can access backend storage.
+###### 1. Admin creates a Bucket API object
 
-### Static Provisioning
-"Driverless" allows the existing workloads to make use of COSI without the need for vendors to create drivers. The following steps show the details of the workflow:
-+ Admin creates `Bucket` instance that references an existing backend bucket.
-+ User creates a `BucketRequest` (BR) that references this `Bucket`.
-+ User creates `BucketAccessRequest` (BAR) that references their BR, a `BucketAccessClass`, and their secret containing access tokens.
-+ COSI detects the existence of the BAR and follows the [grant access](#grant-bucket-access) steps.
-+ COSI detects the existence of the BAR, BA and executes a validation process.
-+ As a validation step, COSI ensures that the `BucketRequest.bucketInstanceName` matches the `BucketAccessRequest.bucketAccessName.bucketInstanceName`. In other words, the two related `Bucket` references match, meaning that the BR and BAR->BA point to the same `Bucket`.
+When a Bucket is manually created, and has its `bucketID` set, then COSI assumes that this Bucket has already been created. 
 
----
+```
+    Bucket - br-$uuid
+    |-------------------------------------------------|
+    | name: bucketName123                             |
+    | spec:                                           |
+    |   bucketID: bucketname123                       |
+    |   protocol: s3                                  |
+    |   parameters:                                   |
+    |     key: value                                  |
+    |   provisioner: s3.amazonaws.com                 |
+    |-------------------------------------------------|
+```
 
-# gRPC Definitions
+###### 2. User creates BucketAccess to generate credentials for that bucket
 
-There is one service defined by COSI - `Provisioner`. This will need to be satisfied by the vendor-provisioner in order to be COSI-compatible
+Unlike the BucketAccess for COSI created bucket, this BucketClaim should directly reference the Bucket instead of a BucketClaim
```

Comment:
Is there a reason why we are not following the PVC/Snapshot model here where you always have a claim even if it's for brownfield? How do we verify that the specified Bucket can satisfy all the fields in the Claim?